membership-attribute-service/conf/routes

```diff
@@ -13,6 +13,9 @@ POST       /user-attributes/me/digitalpack-update-card      controllers.AccountC
 POST       /user-attributes/me/paper-update-card            controllers.AccountController.paperUpdateCard
 POST       /user-attributes/me/contribution-update-card     controllers.AccountController.contributionUpdateCard
 
+POST       /user-attributes/me/contribution-update-dd       controllers.AccountController.contributionUpdateDD
```

Comment:
I know we're following an existing convention here, but is it necessary to have two separate endpoints for this?

Reply:
how do you propose we ensure we target the right product?

it's worth mentioning; in the coming weeks I will be pairing with @jranks123 to have the endpoints accept a sub id. This work was intended to support concurrent recurring contributions, however we could use this idea (of requiring a sub id) to facilitate more generic endpoints

Reply:
Yes, I think we would need to have the client pass a sub (or account) id as part of the request. Let's pick this up separately as it requires some further thought (and probably a refactor of the existing endpoints) which shouldn't block this PR 👍

Reply:
as I say I'll be refactoring all the endpoints with @jranks123 next week anyway, so will hopefully be able to clean up these new endpoints, deprecate lots of the other routes and introduce a few new more generic routes